app/assets/stylesheets/cart.css.scss

```diff
@@ -3,7 +3,7 @@
 }
 
 .cart-picture {
-    background: transparent url("checkout.jpg") center;
+    background: transparent image-url("checkout.jpg") center;
```

Comment:
Color literals like `transparent` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces